README.md

```diff
@@ -2,15 +2,15 @@ This is a user-space file system for interacting with [Google Cloud
 Storage][gcs]. It is not yet ready for public consumption, so please do not use
 it for anything of importance in its current state.
 
-[gcs]: https://cloud.google.com/storage/
+[gcs]: https://cloud.google.com/storage/docs
```

Comment:
Okay to leave as is. I was thinking for the audience, landing on the marketing page is just one more click to get where they want.